src/Graph/GraphPrinter.php

```diff
@@ -101,183 +112,203 @@ class GraphPrinter extends SMWResultPrinter {
 	protected function handleParameters( array $params, $outputmode ) {
 		parent::handleParameters( $params, $outputmode );
 
-		$this->m_graphName = trim( $params['graphname'] );
-		$this->m_graphSize = trim( $params['graphsize'] );
-
-		$this->m_graphLegend = $params['graphlegend'];
-		$this->m_graphLabel = $params['graphlabel'];
-
-		$this->m_rankdir = strtoupper( trim( $params['arrowdirection'] ) );
-
-		$this->m_graphLink = $params['graphlink'];
-		$this->m_graphColor = $params['graphcolor'];
-
-		$this->m_nameProperty = $params['nameproperty'] === false ? false : trim( $params['nameproperty'] );
-
-		$this->m_parentRelation = strtolower( trim( $params['relation'] ) ) == 'parent';
-
-		$this->m_nodeShape = $params['nodeshape'];
-		$this->m_wordWrapLimit = $params['wordwraplimit'];
-
-		$this->m_nodeLabel = $params['nodelabel'];
+		$this->graphName = trim( $params['graphname'] );
+		$this->graphSize = trim( $params['graphsize'] );
+		$this->showGraphLegend = $params['graphlegend'];
+		$this->showGraphLabel = $params['graphlabel'];
+		$this->enableGraphLink = $params['graphlink'];
+		$this->showGraphColor = $params['graphcolor'];
+		$this->rankdir = strtoupper( trim( $params['arrowdirection'] ) );
+		$this->nameProperty = $params['nameproperty'] === false ? false : trim( $params['nameproperty'] );
+		$this->parentRelation =
+			strtolower( trim( $params['relation'] ) ) == 'parent';        // false if anything other than 'parent'
+		$this->nodeShape = $params['nodeshape'];
+		$this->wordWrapLimit = $params['wordwraplimit'];
+		$this->nodeLabel = $params['nodelabel'];
 	}
 
-	protected function getResultText( SMWQueryResult $res, $outputmode ) {
 
+	/**
+	 * @param SMWQueryResult $res
+	 * @param $outputmode
+	 * @return string
+	 */
+	protected function getResultText( SMWQueryResult $res, $outputmode ) {
 		if ( !class_exists( 'GraphViz' )
 			&& !class_exists( '\\MediaWiki\\Extension\\GraphViz\\GraphViz' )
 		) {
 			wfWarn( 'The SRF Graph printer needs the GraphViz extension to be installed.' );
 			return '';
 		}
 
-		$this->isHTML = true;
+		///////////////////////////////////
+		// GRAPH OPTIONS
+		///////////////////////////////////
```

Comment:
Not like this. If the code is not clear try extract well named functions.

Reply:
I removed those heavy comments and if necessary i outsourced code to own function.

See: https://github.com/RacoonDev/SemanticResultFormats/commit/76983e5c31d988a87b94720aee2215e48ae29886